webpack.config.js

```diff
@@ -66,10 +62,8 @@ module.exports = function (env) {
 			path.resolve(__dirname, 'src/migrations')
 		)
 	]
-	if (env == null || !env.noOptimize) {
```

Comment:
This removes the noOptimize option we use in dindctl to debug locally with unminified code. Is there a way to do a similar thing now? Maybe not setting `mode` to "production" if the option is set?

Reply:
Yeah, it only defaults to true in production mode, see https://webpack.js.org/configuration/optimization/#optimization-minimize - you can explicitly disable it even in production mode with
```json
  optimization: {
    minimize: false
  }
```
though.  The `mode: 'production'` vs `mode: 'development'` flag triggers a lot more options for debuggability though so it probably makes sense to switch the mode rather than individual flags: https://webpack.js.org/concepts/mode/

Reply:
Readded support for `env.noOptimize` @pcarranzav